source/avb.rst

```diff
@@ -37,7 +37,10 @@ System Requirements
 -------------------
 
 This tutorial has been validated on two desktop machines with Intel(R) Ethernet
-Controller I210 connected back-to-back and Linux kernel version 4.19.
+Controller I210 connected back-to-back and Linux kernel version 4.19. However,
+only the machine acting as "Talker" needs a TSN capabable Ethernet controller
```

Comment:
I could find 2 typos in this patch. One in the commit message (s/machnine/machine/) and another on in this line (s/capabable/capable/).

Reply:
The listener needs to support gPTP h/w timestamping, right? I guess I think changing this text has the potential to create confusion. It is fine to state the validation setup is two i210s. Actual requirements for a listener should probably go somewhere else completely. Or provide a reference to external documentation.